player.html

```diff
@@ -1,130 +1,154 @@
+                          <p class="desc5">Learn the most effective ways to interact with and teach your child about mordern tech <b>Read More</b></p>
+                        </div>
+                  </div>
+                </a>
+  
+                <a href="">
+                  <div class="suggest">
+                      <img src="images/6. Professional.jpg" height="200" width="300" alt="Professional">
+                        <div class="suggest-text">
+                          <p class="suggest-head"><b>Being Professional</b></p>
+                          <p class="desc6">Learn the best programming languages that could make you a professional web <b>Read More</b></p>
+                        </div>
+                  </div>
+                </a>
+  
+                <a href="">
+                  <div class="suggest">
+                      <img src="images/7. Databases.jpg" height="200" width="300" alt="Servers">
+                        <div class="suggest-text">
+                          <p class="suggest-head"><b>Database Essentials</b></p>
+                          <p class="desc7">Learn different database technologies and decide the best one for your <b>Read More</b></p>
+                        </div>
+                  </div>
+                </a>
+  
+                <a href="">
+                  <div class="suggest">
+                      <img src="images/8. Empower Women.jpg" height="200" width="300" alt="Empower Women">
+                        <div class="suggest-text">
+                         <p class="suggest-head"><b>Women in Technology</b></p>
+                         <p class="desc8">Empower women to join the world of technology, make an impact and prove <b>Read More</b></p>
+                        </div>
+                  </div>
+                </a>
+  
+                <a href="">
+                  <div class="suggest">
+                      <img src="images/9. Networking.jpg" height="200" width="300" alt="Networking">
+                        <div class="suggest-text">
+                          <p class="suggest-head"><b>Networking Essentials</b></p>
+                          <p class="desc9">Learn networking and merge that into your professional career development <b>Read More</b></p>
+                        </div>
+                  </div>
+                </a>
+  
+                <a href="">
+                  <div class="suggest">
+                      <img src="images/10. Cable Handling.jpg" height="200" width="300" alt="Network Cable">
+                        <div class="suggest-text">
+                          <p class="suggest-head"><b>Cable Handling</b></p>
+                          <p class="desc10">Learn the proper way to handle messy network cables to make your work easier <b>Read More</b></p>
+                        </div>
+                  </div>
+                </a>
+            </section>
+        </main>
 
-        <div class="foot">
```

Comment:
Great effort so far. Please try and remove double div declarations for a single element.